components/brave_vpn/browser/connection/brave_vpn_os_connection_api_base.cc

```diff
@@ -24,6 +24,25 @@ namespace brave_vpn {
 
 using ConnectionState = mojom::ConnectionState;
 
+// static
+void BraveVPNOSConnectionAPI::CreateInstance(
```

Comment:
calling this `CreateInstance` is confusing because it doesn't create anything (it doesn't even return anything). I think `Init` or similar would be more appropriate and makes more sense with `manager->Initialized`